program-book-club.html

```diff
@@ -13,5 +13,11 @@ <h1>Book Club Programme</h1>
             <p>The book club meets monthly to share views, exchange ideas, and inspire personal and professional growth.</p> 
             <p>Join our <a href="https://womencodingcommunity.slack.com/archives/C06VB4MH17G" target="_blank">slack channel #prog_book_club</a> to be part of this vibrant community.</p>
         </div>
+        <section class="section" id="What we are reading">
```

Comment:
The ideia here is to create dynamic session. You can check books yml for example and make this dynamic.